bin/index.js

```diff
@@ -86,6 +96,12 @@ module.exports = function(Aquifer, AquiferGitConfig) {
       return false;
     }
 
+    // Validate custom signature options.
+    if (!options.name != !options.email) {
```

Comment:
Strict equals. Additionally, this might be a little confusing with the triple negatives.